machine-learning/03-medulloblastoma_PLIER.Rmd

```diff
@@ -210,22 +208,22 @@ plier_summary_df
 ```
 
 We can filter this `data.frame` based on significance of associations.
-We'll use `FDR < 0.05`.
+We'll use `FDR < 0.05`, where `FDR` is the False Discovery Rate. 
 
 ```{r}
 plier_summary_df %>%
   dplyr::filter(FDR < 0.05)
 ```
 
-We can also sort by AUC values after filtering.
+We can also sort by AUC (Area Under the ROC Curve) values after filtering.
```

Comment:
This might be a good link to go here: https://towardsdatascience.com/understanding-auc-roc-curve-68b2303cc9c5